js/src/hooks/useMCIssuesTypeFilter.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import { ISSUE_TABLE_PER_PAGE } from '.~/product-feed/constants';
+import { ISSUE_TYPE_ACCOUNT } from '.~/constants';
+import useAppSelectDispatch from '.~/hooks/useAppSelectDispatch';
+
+const defaultQuery = {
+	page: 1,
+	per_page: ISSUE_TABLE_PER_PAGE,
+};
+
+/**
+ * Hook that returns the issues data filtered by issueType with their paginator state.
+ *
+ * @param {string} [issueType=product|account] `account` by default
+ * @param {number} page allows to set the page, by default 1
+ * @param {number} perPage allows to set the number of items per page, by default 5
```

Comment:
💅 
Suggest indicating they are optional and have default values.

Reply:
Thanks. Done  here https://github.com/woocommerce/google-listings-and-ads/pull/1176/commits/f7bb1266d37a97d902e7539f74623f6348f582df